env/staging/kustomization.yaml

```diff
@@ -11,6 +11,10 @@ resources:
   - document-download-api-target-group.yaml
   - documentation-target-group.yaml
   - aws-auth-configmap.yaml
+  - celery-sms-send-deployment.yaml
+  - celery-sms-send-service.yaml
+  - celery-sms-send-hpa.yaml
+  - celery-deployment.yaml
```

Comment:
Hopefully this will mean that the copy of `celery-deployment.yaml` in the staging env overrides the one in base?